### PR TITLE
Updated the long running instances list

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description
Skip the instances in the block_list to notify alert. 
Remove the user tag(`@`) in slack.

### CODE REVIEW

**Review by ebattat — APPROVED** [review]
/lgtm

File: cloudsensei/lambda_function.py [hunk 1]
```diff
@@ -189,6 +220,8 @@ def send_email(self, organized_ec2_data):
         This method send email
         :return:
         """
+        if not organized_ec2_data:
+            return 204, "No long running instances"
```
Comment (ebattat) [inline cloudsensei/lambda_function.py:224]: is it the right way to return 204 ?

Comment (athiruma) [reply cloudsensei/lambda_function.py:224]: yes, that's fine.

**Review by krishvoor — APPROVED** [review]
/lgtm

**Pull request merged**